CMakeLists.txt

```
@@ -81,10 +81,13 @@ set(_alpaka_INCLUDE_DIRECTORY "${_alpaka_ROOT_DIR}/include")
 set(_alpaka_SUFFIXED_INCLUDE_DIR "${_alpaka_INCLUDE_DIRECTORY}/alpaka")
 
 # the sequential accelerator is required for the tests and examples
-if(NOT DEFINED alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE AND (alpaka_BUILD_EXAMPLES OR BUILD_TESTING))
-  option(alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE "enable alpaka serial accelerator" ON)
+if(alpaka_BUILD_EXAMPLES OR BUILD_TESTING)
+  if (NOT DEFINED alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE)
+    option(alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE "enable alpaka serial accelerator" ON)
+  elseif(NOT alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE)
+    message(FATAL_ERROR "Serial backend must be enabled for tests and examples: delete CMakeCache.txt and rerun CMake command")
```

AFAIK this is not necessary. I can just turn on the serial backend via `ccmake` or `cmake-gui` and reconfigure.
```suggestion
    message(FATAL_ERROR "Serial backend must be enabled for tests and examples.")
```

---

Not familiar with cmake-gui, but certainly with cmake command line it will not overwrite the already set variable (see also my other comment).

---

@StewMH ONe thing we should test before giving up with overwriting the variable: https://stackoverflow.com/a/39592445

```CMake
set(alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE ON CACHE BOOL "enable alpaka serial accelerator" FORCE)
```

If stack overflow is correct this should work.

---

This seems to work, thanks!